plugin/documents.py

```diff
@@ -489,9 +490,13 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
-            annotations = ["<div class=\"actions\">{}</div>".format(code_actions_link)]
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                title = next(iter(responses.values()))[0]['title']
+                title = "<br/>".join(textwrap.wrap(title, width=30))
```

Comment:
`<br>` was never a self-closing element in the spec. `<br>` is enough. :)

Reply:
Yes,  but in my test, it works and `\n` doesn't work